xbmc/cores/paplayer/PAPlayer.cpp

```diff
@@ -860,6 +860,12 @@ inline bool PAPlayer::ProcessStream(StreamInfo *si, double &freeBufferTime)
       {
         free_space = static_cast<double>(si->m_stream->GetSpace()) *
                      si->m_audioFormat.m_streamInfo.GetDuration() / 1000;
+
+        if (si->m_audioFormat.m_streamInfo.m_type == CAEStreamInfo::STREAM_TYPE_TRUEHD &&
+            !m_processInfo->WantsRawPassthrough())
+        {
+          free_space /= 2;
```

Comment:
This I don't fully get. Reason is: "Our frames are smaller" see below frameSent are divided by two. There I would logically assume that the "freeSpace" is twice as large.
```
free_space *= 2;
```

Reply:
This is not correct. See:

![vs3](https://user-images.githubusercontent.com/58434170/218254263-9bb7f424-cf27-4b89-a50a-43eca812797c.png)


But anyway i can change to form `GetDuration() / 1000 / 2;`  as seems less confusing

Reply:
The confusion comes from the tunnel ... time is only half and therefore space as well ...